testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -335,3 +335,14 @@ def test_missing_endif(self):
         with pytest.raises(IOError):
             with self.parser(ITP_no_endif) as p:
                 top = p.parse(include_dir=GMX_DIR)
+
+class Testrelativepath:
+
+    def test_relpath(self, tmp_path):
+        content="[ atoms ]\n" +\
```

Comment:
It would be neater here to use `"""` multiline strings. I don't think the indentation will matter to the ITPParser.